simpledotorg/simple-server
# Fix for scrolling to top when using toggle

**Story card:** [sc-10766](https://app.shortcut.com/simpledotorg/story/10766/preserve-scroll-position-after-clicking-the-non-contactable-toggle)

## Because

Clicking the toggle causes a page reload and the user is navigated to the top of the page losing their current scroll position.

The user then has to scroll back down to the overdue section.

## This addresses

This corrects the problem and keeps the user in the position they were previously viewing by scrolling to the same position after the pageload

## Test instructions

Scroll down to the overdue section and click the 'non-contactable patients' toggle.
Your page should refresh and the scroll position preserved.
Check both activating and deactivating the toggle.
